ba/tutorials/tour/abstract-types.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: tutorial
+title: Apstraktni tipovi
+
+disqus: true
+
+tutorial: scala-tour
+num: 22
+outof: 33
+language: ba
+---
+
+U Scali, klase su parameterizovane vrijednostima(parameteri konstruktora) i tipovima(ako su [generičke](generic-classes.html)).
+Zbog dosljednosti, ne samo da je moguće imati vrijednosti kao članove objekta već i tipove. 
+Nadalje, obje forme članova mogu biti konkretne ili apstraktne.
+Slijedi primjer koji sadrži obje forme: apstraktnu vrijednost i apstraktni tip kao članove [klase](traits.html) `Buffer`.
+ 
+    trait Buffer {
+      type T
+      val element: T
+    }
+ 
+*Apstraktni tipovi* su tipovi čiji identitet nije precizno definisan.
+U gornjem primjeru, poznato je samo da svaki objekat klase `Buffer` ima a tip-član `T`, 
+ali definicija klase `Buffer` ne kazuje kojem konkretno tipu odgovara `T`. 
+Kao i definicije vrijednosti, možemo prebrisati(override) definicije tipova u podklasama.
```

Comment:
I've replaced it with "redefinisati". It seemed more clear to me but if you have some other suggestion I'll change it.